### PR TITLE
Add support for Imgur

### PR DESCRIPTION
**Pull request by mehtaarn000**

### CODE REVIEW

**Comment by deceitfulvision** [comment]
Fix comma at the end of line 782.

**Comment by mehtaarn000** [comment]
Wait what's wrong with it?

File: sherlock/resources/data.json [hunk 1]
```diff
@@ -773,6 +773,13 @@
     "username_claimed": "adam",
     "username_unclaimed": "noonewouldeverusethis"
   },
+  "Imgur": {
+    "errorType": "No results",
+    "url": "https://www.imgur.com/user/{}",
+    "urlMain": "https://www.imgur.com",
+    "username_claimed": "blue",
+    "username_unclaimed": "noonewouldeverusethis",
```
Comment (deceitfulvision) [inline sherlock/resources/data.json:781]: Remove the comma at the end of this line.

Comment (mehtaarn000) [reply sherlock/resources/data.json:781]: Ohh ok doing it when I'm home

**Comment by sdushantha** [comment]
Imgur had been removed a while ago because it kept giving us false positives.
https://github.com/sherlock-project/sherlock/blob/master/removed_sites.md#imgur

**Pull request closed without merging**